pkg/nsx/services/securitypolicy/firewall_test.go

```diff
@@ -1073,11 +1073,11 @@ func TestCreateOrUpdateSecurityPolicy(t *testing.T) {
 	mockVPCService := common.MockVPCServiceProvider{}
 	fakeService.vpcService = &mockVPCService
 
-	podSelectorRule0IDPort000 := fakeService.buildExpandedRuleId(fakeService.buildRuleID(&spWithPodSelector, &spWithPodSelector.Spec.Rules[0], 0, common.ResourceTypeSecurityPolicy), 0, 0)
-	podSelectorRule1IDPort000 := fakeService.buildExpandedRuleId(fakeService.buildRuleID(&spWithPodSelector, &spWithPodSelector.Spec.Rules[1], 1, common.ResourceTypeSecurityPolicy), 0, 0)
+	podSelectorRule0IDPort000 := fakeService.buildExpandedRuleID(&spWithPodSelector, &spWithPodSelector.Spec.Rules[0], 0, 0, 0, false, -1, common.ResourceTypeSecurityPolicy)
+	podSelectorRule1IDPort000 := fakeService.buildExpandedRuleID(&spWithPodSelector, &spWithPodSelector.Spec.Rules[0], 1, 0, 0, false, -1, common.ResourceTypeSecurityPolicy)
```

Comment:
Why change here to use `spWithPodSelector.Spec.Rules[0]` for `Rule1ID`?

Reply:
That is a mistake, thanks for pointing it out. Corrected.